src/core_plugins/kibana/public/discover/_discover.scss

```diff
@@ -0,0 +1,306 @@
+.dscWrapper {
+  padding-right: 0;
+  padding-left: 21px;
+  z-index: 1
+}
+
+.dscWrapper__content {
+  padding-right: $euiSize;
+  clear: both;
+}
+
+.dscTimechart {
+  display: block;
+  position: relative;
+
+  // SASSTODO: the visualizing component should have an option or a modifier
+  .series > rect {
+    fill-opacity: 0.5;
+    stroke-width: 1;
+  }
+
+  // SASSTODO: Seems that this selector is not used anywhere
+  visualize {
+    height: 200px; /* 1 */
+
+    .loading {
+      opacity: 1 !important;
+    }
+
+    &.only-spy {
+      height: auto;
+
+      .visualize-table {
+        height: auto;
+      }
+    }
+  }
+
+  // SASSTODO: Seems that this class is not used anywhere
+  .chartwrapper {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  // SASSTODO: Seems that this class is not used anywhere
+  .vis-overlay {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.dscTimechart__header {
+  display: flex;
+  justify-content: center;
+  min-height: $euiSizeXXL;
+  padding: $euiSizeXS 0;
+}
+
+.dscOverlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 20;
+  padding-top: $euiSizeM;
+
+  opacity: 0.75;
+  text-align: center;
+  background-color: transparent;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-info {
+  line-height: 30px;
+  padding: 0 10px;
+  border-bottom-left-radius: $euiBorderRadius;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-info-title {
+  font-weight: bold;
+  margin-right: 10px;
+}
+
+.dscTable {
+  overflow: auto;
+
+  // SASSTODO: add a monospace modifier to the doc-table component
+  tbody {
+    font-family: 'Lucida Console', Monaco, monospace;
+  }
+}
+
+// SASSTODO: replace the padding value with a variable
+.dscTable__footer {
+  background-color: $euiColorLightShade;
+  padding: 5px 10px;
+  text-align: center;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-table-details-toggle {
+  margin-bottom: 3px;
+}
+
+.dscTable__timefield {
+  white-space: nowrap;
+}
+
+/**
+ * 1. When switching between open and closed, the toggle changes size
+ *    slightly which is a problem because it forces the entire table to
+ *    re-render which is SLOW.
+ */
+.dscTable__openButton {
+  appearance: none;
+  background-color: transparent;
+  padding: 0;
+  border: none;
+  width: 14px; /* 1 */
+  height: 14px;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-field-filter {
+  background-color: $euiColorLightShade;
+  margin-right: 10px;
+
+  .form-group {
+    margin-bottom: 0;
+  }
+}
+
+/**
+ * 1. Override sidebar-item-title styles.
+ */
+.dscSidebarItem {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 0 !important; /* 1 */
+  padding-bottom: 0 !important; /* 1 */
+  height: $euiSizeXL;
+
+  &--active {
```

Comment:
I think we should probably write them out fully to make search/replace easier. @cchaos ?

Reply:
You're right, besides we use full selector names everywhere. Fixed this one